packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -2517,6 +2526,16 @@ export function attach(
     }
   }
 
+  function forceRerender(id: number) {
+    const fiber = findCurrentFiberUsingSlowPathById(id);
+
+    if (fiber !== null) {
+      if (typeof scheduleUpdate === 'function') {
```

Comment:
Can't this line be merged with the line above?

Reply:
Yup! I've pushed the changes.